Makefile

```diff
@@ -60,7 +60,7 @@ smoketest:
 scaletest:
 	./tests/scalability/scale_test_runner.py -r 10 -d 30
 
-deploy: deploy-chalice deploy-daemons
+deploy: check-secrets deploy-chalice deploy-daemons
```

Comment:
It may desirable, occasionally, to deploy manually without checking secrets, and it should be possible to do this without hacking build files.

Lets move this change to the deploy stages in `.gitlab-ci.yml` and `.gitlab-prod-ci.yml`. This way it's enshrined in our CI/CD process while preserving our ability to perform manual operations.

Reply:
Sounds good!

Reply:
I'd argue that it's not a hack and a necessary dependency if you're deploying.  That said, I think a check in gitlab would probably accomplish the same goal as far as flagging someone to change them back.